csharp/ql/lib/semmle/code/csharp/security/dataflow/ExternalAPIsQuery.qll

```diff
@@ -78,6 +78,8 @@ private module RemoteSourceToExternalApiConfig implements DataFlow::ConfigSig {
   predicate isSource(DataFlow::Node source) { source instanceof ActiveThreatModelSource }
 
   predicate isSink(DataFlow::Node sink) { sink instanceof ExternalApiDataNode }
+
+  predicate observeDiffInformedIncrementalMode() { any() }
```

Comment:
This will also affect the output of `cs/count-untrusted-data-external-api` (if run in diff informed mode). However, this query is not included in any query suite, so this should be ok.

Reply:
Since the same objection as in https://github.com/github/codeql/pull/19957#discussion_r2211563225 applies, I'll remove this commit.